src/components/UiText/index.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import styled from 'styled-components';
+
+const VARIANTS = {
+  h1: 'h1',
+  h2: 'h2',
+  h3: 'h3',
+  h4: 'h4',
+  h5: 'h5',
+  h6: 'h6',
+  content: 'content',
+  contentBold: 'contentBold',
+  subContent: 'subContent',
+  subContentBold: 'subContentBold',
+};
+
+const variants = Object.keys(VARIANTS);
+
+const styles = {
+  fontSize({ variant }) {
+    if (variant === VARIANTS.h1) {
```

Comment:
:tulip:  Do you wanna enforce this style of conditionals? I find objects to be way more concise

```javascript
const variants = {
[VARIANTS.h1]: 'var(--f-fontSize--xxl)',
[VARIANTS.h2]: 'var(--f-fontSize--xl)'
.....
}

return variants[variant] || undefined 
```
What do you think?

Reply:
This is close to how Ugo did it right? Yeah I think it's cleaner for components such as UiText, that use a lot of `if`s.

Reply:
For other components I think using `if`s is better. Looking at the `button` there is some logic such as `if (variant === VARIANTS.OUTLINE && intent === INTENTS.PRIMARY)` that would be annoying to implement in object syntax

Reply:
👍 I really like this when it makes sense: https://github.com/heetch/flamingo/pull/110/commits/5784e8e262def6dce964a97fbf902bb8aba8a29a#diff-4589e5744b98bb9a842c6b69878e29a9